content/blog/2020/04/2020-04-16-github-app-authentication.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "GitHub app authentication released"
+tags:
+- github
+- github-branch-source
+author: timja
```

Comment:
Some social media image would be great. E.g we could simply emulate the GitHub Web UI:

![image](https://user-images.githubusercontent.com/3000480/79386750-e3015e00-7f6a-11ea-8fea-6e15b5b3a8f6.png)


Source: https://docs.google.com/presentation/d/1ncGvxS3KX1JesCg9FJ04y94jdEk9mLKWbbVwBUbyPuo/edit?usp=sharing